build.gradle

```diff
@@ -12,5 +12,8 @@ buildscript {
 allprojects {
     repositories {
         jcenter()
+        maven {
+            url 'http://dl.bintray.com/raphaelbussa/maven'
```

Comment:
I do not think @chrisbanes will like this. Just upload your project to `jcenter()`.